extmod/machine_mem.c

```diff
@@ -3,7 +3,7 @@
  *
  * The MIT License (MIT)
  *
- * Copyright (c) 2013, 2014 Damien P. George
+ * Copyright (c) 2013, 2014, 2015 Damien P. George
```

Comment:
I've been wondering about this for a while: is the date really necessary (because copyright isn't timeless or so?), and is the plan to continue to just add years?